client/src/components/Main/dummyData/dummyData.js

```diff
@@ -0,0 +1,312 @@
+export default [
```

Comment:
why do we need this file? Is the other dummy data not sufficient?

Reply:
There are multiple versions of dummy data defined as consts in test files all over client.
There is no central location on client to pull this data from. Maybe we should raise as an issue to refactor tests to take dummyData from a central point?